### ISSUE TITLE
Way to specify 'the rest'?

### ISSUE THREAD

**Issue by mcblum** [sirlantis/gulp-order#22]
Trying to get jQuery to load first so the rest of its dependencies will work. The issue is I'm using this in conjunction with main-bower-files. Is there a way to say: load jQuery first and then put everything else after that?

Thank you!
Matt

**Comment by Nate-Wilkins** [comment]
I'm doing something very similar.

```js
var dependencies = gulp.src(mainBowerFiles('**/*'), { read: false })
    .pipe(order([
      'normalize.css'
    ]));
```

trying to get normalize to order first while respecting the original sort order, which doesn't seem to be working... Any progress on this?

**Comment by Gobd** [comment]
```
gulp.task('js', function() {
    return gulp.src(mainBowerFiles('**/*.js').concat(['src/**/*.js'])
        .pipe(order([
            "**/jquery.js", "**/app.js", "**/*.js"
        ]))
        .pipe(uglify())
        .pipe(concat('min.js'))
        .pipe(gulp.dest('./dist/js'))
        .on('end', reload);
});
```

That should work for you guys. Let me know if you're still interest/have any problems with it and I can offer some help. It took me a while to figure it out on my own.

**Comment by Nate-Wilkins** [comment]
So in my example it would look something like
```js
var dependencies = gulp.src(mainBowerFiles('**/*'), { read: false })
    .pipe(order([
      '**/normalize.css',
      '**/*'
    ]));
```

**Comment by Gobd** [comment]
Yes that would probably work, I've never done it without file extensions but it looks good.

**Comment by Nate-Wilkins** [comment]
Yeah that worked, thanks!

**Comment by arafat-amin** [comment]
Hi guys, hi @Gobd 
This way does not work as I expected, so anyone can lend a hand please:

`
function minifyJS() {
    return src('src/js/**/*.js')
    .pipe(order([
        'src/js/jquery.js',
        'src/js/**/*.js',   
        'src/js/script.js' 
    ]))
        .pipe(concat('bundle.js'))
        .pipe(terser())
        .pipe(dest('dist/js'));
} 
`
